roles/ssp-modules/defaults/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+
+
+ssp_mods_extra_enabled: []
+ssp_modules_db: []
+
+
+ssp_adminpassword_key: auth.adminpassword
+ssp_secretsalt_key: secretsalt
```

Comment:
In this necessary?

Reply:
You are right! It isn't necessary! We do not need them here - it's my fault.